contracts/Orchestrator.sol

```diff
@@ -47,7 +47,7 @@ contract Orchestrator is Ownable {
         for (uint256 i = 0; i < transactions.length; i++) {
             Transaction storage t = transactions[i];
             if (t.enabled) {
-                bool result = externalCall(t.destination, t.data);
+                (bool result, bytes memory reason) = t.destination.call(t.data);
```

Comment:
Seems like we're not using `reason`, so maybe not reference it?

```
 (bool result, ) = t.destination.call(t.data);
```

Reply:
yeah good call I'll remove it